libcloud/httplib_ssl.py

```diff
@@ -149,7 +149,12 @@ def _setup_ca_cert(self):
         if self.verify is False:
             pass
         else:
-            self.ca_cert = libcloud.security.CA_CERTS_PATH
+            if isinstance(libcloud.security.CA_CERTS_PATH, list):
+                if len(libcloud.security.CA_CERTS_PATH) > 1:
+                    raise ValueError('Only 1 certificate path is supported')
```

Comment:
I guess this was intended to be a warning? The next line will never be executed.

When removing the raised exception, I get a 'Not such file a directory' exception, unless the only line left in CA_CERTS_PATH is a path that exists on my machine. When it's a path that does not exist, the exception is raised by [ssl.SSLContext.load_verify_locations](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_verify_locations) which is called by urllib3 which is called by requests.